src/commands/impl/playlist.ts

```diff
@@ -23,15 +23,21 @@ const playlist: CommandBinder = (client: Client) => ({
             }
             const defaultRange = {start: dayStart, end: now};
             const range = await RangeDeterminer.determineRange(client, message, remainingArgs, defaultRange);
-            await createPlaylist(source, message, range);
+            const existingPlaylist = await database.getPlaylist(getGuild(message), source, range);
```

Comment:
The range here is the wrong range! I need to use the range of the songs, not the range requested.